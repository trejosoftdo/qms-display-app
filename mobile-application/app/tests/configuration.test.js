import React from 'react';
import Configuration from '../configuration';


describe('Configuration route component', () => {
  it('renders the component as expected', () => {
    expect(renderShallow(<Configuration  />)).toMatchSnapshot();
  });
});
