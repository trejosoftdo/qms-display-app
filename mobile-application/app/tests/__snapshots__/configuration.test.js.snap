// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Configuration route component renders the component as expected 1`] = `
<React.Fragment>
  <AppHeader
    title="Translated[translation:configuration]({})"
  />
  <ConfigurationView />
</React.Fragment>
`;
